src/vm/amd64/asmconstants.h

```diff
@@ -62,11 +62,19 @@ ASMCONSTANTS_C_ASSERT(ASM_CLRTASKHOSTED   == CLRTASKHOSTED);
 #define PINVOKE_CALLI_SIGTOKEN_REGNUM        11
 #define PINVOKE_CALLI_SIGTOKEN_REGISTER     r11
 
+#ifdef UNIX_AMD64_ABI
+// rdi, rsi, rdx, rcx, r8, r9
+#define SIZEOF_MAX_OUTGOING_ARGUMENT_HOMES  0x30
```

Comment:
The Unix calling convention does not have outgoing argument homes. For Unix, I would call this constant SIZEOF_MAX_INT_ARG_SPILL - to follow the naming convention used in this file.